test/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrCodeflowTest.cs

```diff
@@ -422,6 +422,195 @@ public async Task OutOfOrderMergesTest()
         await GitOperations.CheckAllIsCommitted(ProductRepoPath);
     }
 
+    // This repo simulates frequent changes in the Version.Details.xml file.
+    // It tests how updates to different packages would (not) conflict with each other.
+    [Test]
+    public async Task VersionDetailsConflictTest()
+    {
+        const string branchName = nameof(VersionDetailsConflictTest);
+
+        await EnsureTestRepoIsInitialized();
+
+        await File.WriteAllTextAsync(ProductRepoPath / VersionFiles.VersionDetailsXml,
+            """
```

Comment:
@mmitche so this works. I originally though it doesn't because I included empty lines and the XML serialization was removing those and those were the conflicts and not the lack of context. Below is also a `Versions.props` file. It's possible to play with the paddings here and run the test to quickly tell what padding is sufficient.